tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -266,6 +281,22 @@ def _test_storage_type(storage_type, fsx_fs_id, region):
         assert_that(get_storage_type(fsx_fs_id, region)).is_equal_to("SSD")
 
 
+def get_deployment_type(fsx_fs_id, region):
+    logging.info("Getting DeploymentType from DescribeFilesystem API.")
```

Comment:
What about logging this message when we have the value and log the deployment type too?

Reply:
Done